packages/atomic/src/components/search/atomic-relevance-inspector/atomic-relevance-inspector.tsx

```diff
@@ -22,8 +23,15 @@ export class AtomicRelevanceInspector {
   closeRelevanceInspector: EventEmitter<null> | undefined;
 
   public render() {
-    const {platformUrl, organizationId} =
+    const {organizationId, platformUrl} =
       this.bindings.engine.state.configuration;
+    const envMatch = platformUrl.match(/(hipaa|dev|stg)/);
```

Comment:
If somebody uses organization URLs and their organization happens to contain `hipaa`, `dev` or `stg` in the name, I think this would get matched?

I think this needs a more strict regular expression. Alternatively, Headless could provide a way of getting the current admin endpoint OOTB.

Reply:
> Alternatively, Headless could provide a way of getting the current admin endpoint OOTB.

It does, that's what the `getOrganizationEndpoints` utility function does. We still need to pass it the environment though

Reply:
I meant that it could be stored in the reducer just like the `platformUrl` and `analytics.apiBaseUrl` (even though we may not use it in Headless *yet*).

Reply:
Ok, in this particular case I don't think it's a good idea to store that value in Headless, it's pretty corner case that we ourselves use it; most clients won't have any need for that value when building any UI with Headless.